### DISCUSSION TITLE
Use of Allan Variance example at 1Hz sample rate

### DISCUSSION THREAD

**Discussion by amatsaka** [rodralez/NaveGo#57] [Q&A] [answered]
Hi, 
I am trying to use the Allan-Variance example with my set of data sampled at 1Hz. 

This gives an error at allan_overlap.m line 196 "Either "time" or "rate" must be present in DATA" because the calculated data.rate at allan_imu.m line 153 is zero. 
Is there any limitation in sampling rate?

Best Regards!!

**Comment by rodralez** [comment, marked as answer]
Dear amatsaka,

You just have found a bug in NaveGo's code. This message should have had been an issue.

The problem was in function allan_imu(), line 151. When frequency was 1 Hz, it was rounded to zero.

Please, download NaveGo's last commit and let me know if everything is OK.

Best regards.
